using-timescaledb/distributed-hypertables.md

```diff
@@ -48,32 +54,203 @@ CREATE TABLE conditions (
+  number of space partitions was correspondingly increased), then
+  a full push down might not be possible when it otherwise would
+  be. In our example distributed hypertable, the change in
+  partitioning might lead to data for the same location existing in
+  two chunks that are stored on different data nodes. The planner must
+  therefore avoid *full* push down whenever the query covers the
+  repartitioning boundary. A *partial* push down is still possible,
+  however.
+- If the query includes non-immutable functions and expressions, the
+  planner cannot push down those parts of the query since they cannot
+  be guaranteed to generate a consistent result across each data
+  node. For instance, the [`random()`][random-func] function is
+  [`VOLATILE`][volatility] since it depends on the current seed and
+  the state of the pseudo-random sequence. If the function was to be
+  pushed down to each data node, it would not generate a valid
+  pseudo-random sequence from the point of view of the access node
+  that executes the query. Another example is the
+  [`now()`][current_time] function to get the current transaction
+  time. This is a [`STABLE`][volatility] function that depends on the
+  current time zone setting on each node. TimescaleDB specifically
+  optimizes the use of `now()`, however, by constifying the function
+  on the access node, thus it is able to push down the resulting
+  timestamp to the data nodes, but this is not the case for many
+  `STABLE` functions.
+- If the query includes a user-defined function (UDF) the planner
+  assumes that the function does not exist on the data nodes and
+  therefore will not push it down.
+
+## Data Node Management [](data-node-management)
+
+When a distributed hypertable is created, it will use all the
+available data nodes by default. It is possible, however, to use only
+a subset of data nodes with a particular distributed hypertable. This
+is useful to, e.g., tie a distributed hypertable to data nodes that
+have a specific performance profile.
+
+If additional data nodes are added to a distributed database, the data
+nodes are not automatically associated with existing distributed
+hypertables. Instead, you need to explicitly *attach* a data node
+using [`attach_data_node`][attach_data_node]:
 
 ```sql
 SELECT add_data_node('node3', host => 'dn3.example.com');
 SELECT attach_data_node('node3', hypertable => 'conditions');
 ```
 
-Once the data node has been attached, any new chunks will also be
-distributed to the hypertable on the new data node.
+When attaching a data node, the partitioning configuration of the
+distributed hypertable is also updated to account for the additional
+data node (i.e., the number of space partitions is automatically increased to
+match), unless the function parameter `repartition` is set to
+`FALSE`. The updated partitioning configuration ensures that the data
+node will be able to take on newly created chunks.
```

Comment:
It is not very clear that the new data node will only be used with new chunks, that is, it will not re-partition existing chunks.

Reply:
I am intentionally not mentioning it here since it is only a current limitation. We're mentioning elsewhere that this limitation exists.